### PR TITLE
release 0.5.0

### PR DESCRIPTION
**Pull request by Melkiades**
[skip vbump] [skip spelling]

#149

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
- * new `diagnose_pagination` function which accepts the same arguments as `paginate_to_mpfs` and returns information about the pagination algorithm outcome at both row and column positions.
- * `MatrixPrintForm` objects now track referential footnote information much more carefully than before
- * `MatrixPrintForm` objects will infer detailed referential footnote information from their `strings` element for backward compatibility.
- * fix to test that failed on old Windows CRAN machine due to imperfect UTF support there.
- * when specifying `lpp` and `cpp` in pagination or exporter functions, `NULL` now unambigiously means no pagination in the relevant direction, while the new default of `NA_integer_` now means those values should be inferred from page size.
-
-## formatters 0.4.1.9001
+## formatters 0.5.0
+ * `font_size` is now 8 consistently across the pagination machinery.
+ * Margins are interpreted in inches when specified (including the default).
```
Comment (Melkiades) [inline NEWS.md:3]:
```suggestion
 * When specified, margins are in inches when specified (including the default).
```

File: NEWS.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+ * Fix off-by-one error in pagination machinery for certain cases.
+ * Refined informative messages when pagination is performed with `verbose = TRUE`.
```
Comment (Melkiades) [inline NEWS.md:5]:
```suggestion
 * Refined informative messages when pagination is verbose (`verbose = TRUE`).
```

File: NEWS.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
+ * Referential footnotes with specified (non-number) symbols are now printed once per page regardless of how many elements on the page they refer to.
```
Comment (Melkiades) [inline NEWS.md:6]:
```suggestion
 * Referential footnotes with specified (non-number) symbols are now printed once per page regardless of the number of elements on the page they refer to.
```

**Review by shajoezhu — APPROVED** [review]
Thanks so much @gmbecker and @Melkiades , downstream are all checked out. we are all good to go

**Pull request merged**